src/app/chat/view.nim

```diff
@@ -853,3 +898,29 @@ QtObject:
     except Exception as e:
       error "Error removing user from the community", msg = e.msg
 
+
+  proc requestToJoinCommunity*(self: ChatsView, communityId: string, ensName: string) {.slot.} =
+    try:
+      let requests = self.status.chat.requestToJoinCommunity(communityId, ensName)
+      for request in requests:
+        self.myCommunityRequests.add(request)
+    except Exception as e:
+      error "Error requesting to join the community", msg = e.msg
+
+  proc acceptRequestToJoinCommunity*(self: ChatsView, requestId: string): string {.slot.} =
+    try:
+      self.status.chat.acceptRequestToJoinCommunity(requestId)
+      self.activeCommunity.communityMembershipRequestList.removeCommunityMembershipRequestItemFromList(requestId)
+    except Exception as e:
+      error "Error accepting request to join the community", msg = e.msg
+      return "Error accepting request to join the community"
```

Comment:
Should we change these error messages to be a key that can be used with trId() on QML?

Reply:
That could be a good idea, but I'm not sure how it would work, since we usually want to add a context comment `//%` to help the translators know what they are translating